src/lib/requests/common.js

```diff
@@ -78,11 +78,26 @@ const throwOrJson = async (res) => {
     return;
 };
 
+const bodyStringifier = (obj) => {
+    if (typeof obj === 'string' || Buffer.isBuffer(obj))
+        return obj;
+    if (typeof obj === 'number')
```

Comment:
is a need for a special case for a number? JSON.stringify should handle the case - or maybe is there any edge case?

Reply:
Yeah I don't know the intent of the original authors. I copied this out of the original `mojaloop-requests.js` file,  but I didn't look too hard at it.

Reply:
It is a low hanging fruit... ;-)